dev/customer_testing/run_tests.dart

```diff
@@ -231,7 +243,7 @@ class TestFile {
     if (update.isEmpty)
       throw FormatException('${errorPrefix}No "update" directives specified. At least one directory must be specified. (It can be "." to just upgrade the root of the repository.)');
     if (test.isEmpty)
-      throw FormatException('${errorPrefix}No "test" directives specified. At least one command must be specified to run tests.');
+      throw FormatException('${errorPrefix}No "test" directives for this platform specified. At least one command must be specified to run tests on each of Windows, MacOS, and Linux.');
```

Comment:
Former-English major nit: 'No "test" directives specified for this platform' could be clearer for readers.